### PR TITLE
Update ONNX Export for Gather and Scatter for Opset 11

### PR DESCRIPTION
**Pull request by lara-hdr**

### CODE REVIEW

**Review by houseroad — APPROVED** [review]
LGTM

File: torch/onnx/symbolic_opset11.py [hunk 1]
```diff
@@ -26,6 +26,16 @@ def pixel_shuffle(g, self, upscale_factor):
     return g.op("DepthToSpace", self, blocksize_i=upscale_factor, mode_s="CRD")
 
 
+@parse_args('v', 'i', 'v', 'v')
+def gather(g, self, dim, index, sparse_grad=False):
```
[the remark this answers is not in the document]
Comment (houseroad) [reply torch/onnx/symbolic_opset11.py]: Nit: can we error out if sparse_grad is True?

**Comment by lara-hdr** [comment]
@ houseroad, can we merge this PR?

**Pull request closed without merging**